Making mediborgs not suck (as much)

Road-Train opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
Removes diagnosis module (unused, functionality moved to rescue module). Adv. chem module now gets automenders. Syndieborg loses rescue module in favor of GUN.

## Why we need to add this
Diagnosis module is left over from a time where not every medical module had an analyser and should've been deleted a long time ago. Unbroken Eclipse wanted me to give borgs automenders. And the syndie medborg gets to have a gun. As a treat. Not like you're gonna defib a puddle of acid.

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [X] I do not require assistance to complete the PR.
- [] Before posting/requesting review of a PR, I have verified that the changes work. (impossible as most recent dev IS BRICKED)
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

:cl: RoadTrain
- add: Adv. chem module now gets automenders
- remove: Removed diagnosis module (stethoscope is in the rescue module). It wasn't being used anyway and was literally useless.
- tweak: Syndie mediborg loses rescue module, gains weapon module.

Road-Train committed:
Removes diagnosis module (unused, functionality moved to rescue module). Adv. chem module now gets automenders. Syndieborg loses rescue module in favor of GUN.

Road-Train committed:
Merge branch 'starlight-dev' of https://github.com/ss14Starlight/space-station-14 into making-mediborg-not-suck

Happyrobot33 commented on Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml:
```diff
@@ -671,21 +671,21 @@
-    - ClothingNeckStethoscope
-  - type: BorgModuleIcon
-    icon: { sprite: Interface/Actions/actions_borg.rsi, state: diagnosis-module }
+# - type: entity
```
dont comment this out, just leave it

Road-Train replied:
I specifically commented out because the module is a leftover from when medical borg didn't have health analyzer in every module. It serves no purpose, not even for admemes. I'd rather it not get spawned in accidentally but if you insist, I can.

Road-Train replied:
Gave it a suffix instead.

Road-Train committed:
Update borg_modules.yml

Uncommented the diagnose module, gave it a suffix instead so people don't spawn it into events

Road-Train committed:
Merge branch 'starlight-dev' into making-mediborg-not-suck
